js/app/restaurant/restaurant.service.js

```diff
@@ -0,0 +1,12 @@
+((app) => {
+	const restaurantService = (restaurantProvider) => {
+		const getRestaurants = () => restaurantProvider.getRestaurants();
+
+		return {
+			getRestaurants,
+		};
+	};
+
+	app
+		.factory('rwApp.restaurantService', ['rwApp.restaurantMockProvider', restaurantService]);
```

Comment:
The disconnect between ```app``` and ```factory``` seems a little extreme given that there are no other chain methods listed here.

Reply:
resolved